apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -0,0 +1,21 @@
+import {expect} from '../../../util/configuredChai';
+import {sortRows} from '@cdo/apps/templates/manageStudents/ManageStudentsTable';
+import {ROW_TYPE} from '@cdo/apps/templates/manageStudents/manageStudentsRedux';
+
+describe('ManageStudentsTable', () => {
+  it('sortRows orders table in the following order: add, newStudent, student', () => {
+    const rowData = [
+      {id: 1, name: 'studentb', rowType: ROW_TYPE.student},
+      {id: 3, name: 'studenta', rowType: ROW_TYPE.student},
+      {id: 0, name: '', rowType: ROW_TYPE.add},
+      {id: 2, name: 'studentf', rowType: ROW_TYPE.newStudent}
+    ];
+    const columnIndexList = [];
+    const orderList = ["asc"];
+    const sortedList = sortRows(rowData, columnIndexList, orderList);
+    expect(sortedList[0].id).to.equal(0);
+    expect(sortedList[1].id).to.equal(2);
+    expect(sortedList[2].id).to.equal(1);
+    expect(sortedList[3].id).to.equal(3);
+  });
```

Comment:
I'm confused - this test makes it seem like the new student being added will go immediately after the add new student row, effectively at the top of the table. When brent and brad are added in the PR description, they're at the bottom of the list. I don't think it matters either way, I just want to double check the test matches the behavior and clarify if I'm reading it incorrectly.

Reply:
Oh... this is before you hit save - so the rows that are actively being edited show up at the top of the table.

Reply:
Yep, this is before hitting save. We want to pin them there because it'd be weird to have them scattered throughout the table. Order is clarified in the comments of sortRows function.